tests/tests_qdevice_tls_kaptb_options.yml

```diff
@@ -0,0 +1,70 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Test qdevice - tls and KAPTB options
+  hosts: all
+  vars_files: vars/main.yml
+  vars:
+    _required_capability: corosync.quorum.device.model.net.options_tls_and_kaptb
+
+  tasks:
+    - name: Set qnetd address
+      set_fact:
+        __test_qnetd_address: "{{
+            (ansible_play_hosts_all | length == 1)
+            | ternary('localhost', ansible_play_hosts[0]) }}"
+
+    - name: Fetch versions of cluster components
+      include_tasks: tasks/fetch_versions.yml
```

Comment:
In every other case, fetch_versions is called after calling the role - fetch_versions needs `pcs` which is installed by the role

Reply:
Yeah, I missed that. Well, in this case, the test needs to know pcs version beforehand, to figure out if it even makes sense to run. I pushed updated commits.